packages/rancher-cis-benchmark/package.yaml

```diff
@@ -1,6 +1,6 @@
 url: local
-packageVersion: 00
-releaseCandidateVersion: 01
+packageVersion: 02
+releaseCandidateVersion: 00
```

Comment:
@aiyengar2 @cbron We are changing rancher-cis-benchmark chart to release version 1.0.4 in this release 2.5.8 - Are the value for packageVersion and releaseCandidateVersion correct?

Reply:
there is no upstream chart for CIS v2, it is a local or rancher original chart.

Reply:
There are multiple changes coming in for the rancher-cis-benchmark chart in this release, hence I wanted to check what should be the values for these. Will each PR need to change the packageVersion?

Reply:
We should only bump the packageVersion when absolutely necessary IMO. If it's possible to bump the patch version instead of the packageVersion, let's do that.

Some reasons why we cannot bump the patch version include:
- The chart is forked from an upstream, so we don't have control over the patch version
- We want to keep the chart version in line with the appVersion

For CIS, I think the latter case applies, so it may make sense to bump the packageVersion instead of the chart version's patch version.

Reply:
ok so if multiple PRs are going in for the new appVersion version to be released in 2.5.8 (1.0.4 is to be released), should each PR change the packageVersion? or do we need one final packageVersion value?

Reply:
To be clear, we only need to bump the packageVersion once per release though. Once you bump it once in a release, you should always only bump the releaseCandidateVersion in any future changes.

Reply:
Just talked to @prachidamle offline about this. Since the latest version released today in https://github.com/rancher/charts/tree/release-v2.5/charts/rancher-cis-benchmark/rancher-cis-benchmark is 1.0.301 and we have already bumped the chart version to 1.0.4, the package.yaml should look like:
```
packageVersion: 00
releasedCandidateVersion: 0X
```

Where we always bump the releasedCandidateVersion only within the context of this release.

This would result in the final shipped package being 1.0.400, which is exactly what we want.

Reply:
@MonzElmasry so we should change this to be:
```
packageVersion: 00
releasedCandidateVersion: 02
```

Reply:
And then 1.0.400 is what we put in the table of that github issue

Reply:
okay I'll create another pr that changes the versions.

Reply:
@aiyengar2 Do we need to delete the pushed chart artifacts or will a new PR correcting the ```releasedCandidateVersion: 02``` regenerate them?

Reply:
but should we manually delete the pushed chart ? @aiyengar2 @cbron